src/content/team-members/nikolay-volodin.md

```diff
@@ -0,0 +1,39 @@
+---
+path: /team/nikolay-volodin
+type: team
+published: 1
+name: Nikolay Volodin
+personal_pronouns: 'he/him'
+audio: 'audio/nikolay-volodin.mp3'
+pronunciation: 'nee-koh-lie voh-loh-din'
+image: 'images/nikolay-volodin.jpg'
+role: Engineering
+location: Ottawa, Ontario
+quote: "I like to see the work I do go from being a ‘task’ to a ‘tangible thing.’ It’s rewarding to see a live website spring forth from just an idea."
+social: 
+  - {
+  name: LinkedIn,
+  url: https://www.linkedin.com/in/nikolay-volodin/
+  }
+  - {
+  name: GitHub,
+  url: https://github.com/klimp-drupal
+  }
+
+specialties: [ "Drupal 8",
+Clean Code,
+Pantheon,
+Devoted Cat Dad
+]
+  
+---
+
+### More about Daniel
```

Comment:
@melindaburgess noticed this minor typo, should be 'More about Nikolay'.

Reply:
@dmundra Thank you for catching that!